packages/cli/src/commands/generate/cell/__tests__/__snapshots__/cell.test.js.snap

```diff
@@ -14,7 +14,7 @@ export const Loading = () => <div>Loading...</div>
 export const Empty = () => <div>Empty</div>
 
 export const Failure = ({ error }) => (
-  <div style={{ color: 'red' }}>Error: {error.message}</div>
+  <div style={{ color: 'red' }}>Error: {error?.message}</div>
```

Comment:
True true

Reply:
I'm not going to change this one actually DT - the only reason the error is optional is because of the Apollo typing - I don't even know in what case error might be null. In 99% of cases, errror will be there... so I think no need to change it.